### PR TITLE
wb: edit intensity slider

### LINKED ISSUE #181: NSF - Exercises 2 & 3: Western Blotting exposure time slider going off the window in its two extreme positions
The exposure time slider when positioned either in the extreme left or right position, is off the demarcated region window where the slider resides.  

screen shot 2015-04-27 at 2 57 56 pm

### PR DESCRIPTION
**Pull request by annagav**
Fix #181 (182, 183)

### CODE REVIEW

**Comment by pdpinch** [comment]
The slider works better now, but should I be seeing the intensity of the blots change (or is that in a different PR)?

For example, here's what I'm seeing at 1 sec and at 1h, no apparent difference:

screen shot 2015-07-23 at 12 11 31 pm

screen shot 2015-07-23 at 12 11 54 pm

**Comment by annagav** [comment]
Change of intensity is fixed in #267.

File: html_app/ui/WesternBlotGelView.js [hunk 1]
```diff
@@ -513,11 +513,6 @@ scb.ui.WesternBlotGelView = function scb_WesternBlotGelView(gstate) {
 				  'aria-valuenow': "0",
 				  'aria-valuetext': "1 minute"
 				});
-				$('.scb_f_wb_exposure_slider > .ui-slider-range').width( $('.scb_f_wb_exposure_slider > .ui-slider-range').width() - 6);
-				if($('.scb_s_wb_exposure_time_value').text() == '1 h') {
-					$('.ui-slider-handle').css('left', $('.ui-slider-handle').position().left-10+'px')
-					console.log('fix');
-				}
```
Comment (pdpinch) [inline html_app/ui/WesternBlotGelView.js:520]: I love seeing code like this removed!

**Comment by pdpinch** [comment]
In that case, :+1:

**Pull request merged**